sdk/java-sdk-spring/src/test/java/kalix/javasdk/client/ComponentClientTest.java

```diff
@@ -251,6 +251,23 @@ public void shouldReturnDeferredCallForPOSTWithRequestParams() throws InvalidPro
     assertThat(getBody(targetMethod, call.message(), Message.class)).isEqualTo(body);
   }
 
+  @Test
+  public void shouldReturnDefferedCallWithTraceparent() {
```

Comment:
```suggestion
  public void shouldReturnDeferedCallWithTraceParent() {
```

Reply:
:) thanks. We almost got it right. I think it is with two `r`.